brut.apktool/apktool-lib/src/main/java/brut/androlib/Androlib.java

```diff
@@ -33,9 +33,11 @@
 import java.nio.file.Path;
 import java.util.*;
 import java.util.logging.Logger;
+import java.util.zip.CRC32;
 import java.util.zip.ZipEntry;
 import java.nio.file.Files;
 import java.util.zip.ZipFile;
+import java.util.zip.ZipOutputStream;
```

Comment:
Can you remove the 3 `java.nio` imports? It seems there are no longer any nio operations in this file.